manifests/params.pp

```diff
@@ -7,6 +7,10 @@
   $service_restart_exec = $::osfamily ? {
     'Debian'  => '/sbin/ifdown -a --force ; /sbin/ifup -a',
     'Solaris' => '/usr/sbin/svcadm restart svc:/network/physical:default',
+    'RedHat'  => versioncmp($facts['os']['release']['major'], '8') ? {
```

Comment:
This is ok for recent Puppet versions but not good for the old versions that this module still supports.
I'd use here the legacy operatingsystemmajrelease fact and use facts only on the next version of the module (planned somewhen, with Puppet > 4 support)